scripts/amd/gemm/tune_gemm.py

```diff
@@ -409,7 +414,7 @@ def profile_batch_kernels(M, N, K, gpuid, gpus, jobs, verbose):
         kernel_name = generated_kernel_name(M, N, K, jobId)
         if verbose:
             print(f"profiling {kernel_name} on GPU {gpuid}")
-        run_bash_command_wrapper(f"rocprof --stats -o results-{jobId}.csv python {kernel_name}", capture=(verbose < 2))
+        run_bash_command_wrapper(f"rocprofv2 --plugin file --plugin-version 1 --kernel-trace -o {jobId} python {generated_kernel_name(M, N, K, jobId)}", capture=(verbose < 2))
```

Comment:
generated_kernel_name(M, N, K, jobId) is the same as `kernel_name`, the latter is simpler.
And why did you change the output from results-{jobId}.csv to {jobId}? Is it automatically expanded to results-{jobId}.csv?

Reply:
yes, it is automatically expand.